submissions/aderonkeOdebunmi/introductionToHtml/productPage.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
```

Comment:
You have this already. That's your `index.html`. You should remove this file as well.